### ISSUE TITLE
error messages are not accessible

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#255]
The way to correct it is described in the report:
![image](https://cloud.githubusercontent.com/assets/7644740/4441365/fd6329f8-47d1-11e4-8ff4-b48978c442b2.png)
